examples/cull-idle/Cull_service_UI/web_service.py

```diff
@@ -0,0 +1,132 @@
+"""
+Task :  Web service for Cull UI
+
+Date : 16 Aug 2016
+
+Version : 1.0
+
+Author : Vigneshwer
+"""
+
+# Loading python modules 
+import tornado.ioloop
+import tornado.web
+import tornado.escape
+from tornado.concurrent import Future
+from tornado.gen import coroutine
+from tornado.options import define, options, parse_command_line
+from tornado.log import app_log
+from tornado.httpclient import AsyncHTTPClient, HTTPRequest
+import tornado.websocket
+import config
+import datetime
+import json
+import os
+import datetime
+from dateutil.parser import parse as parse_date 
+import json
+
+define("port", default=8888, help="run on the given port", type=int)
+define("debug", default=False, help="run in debug mode")
+define("static_path", default="./", help="Content of the template")
+define('url', default='https://127.0.0.1:443/hub', help="The JupyterHub API URL")
+
+# Values to be changed
+api_token="c4297f02c25b49c2a63c9b102b24cc51"
+url = 'https://127.0.0.1:443/hub'
+certi_loc='./certs/jupyterhub.crt'
+cull_time=1000
+
+# Constant variables
+clients = [] 
+session_info={}
+time_out_val=None
+
+# Renders the index page
+class MainHandler(tornado.web.RequestHandler):
+	@coroutine
```

Comment:
Looks like you have indented with tabs. The common convention in Python is to indent with four spaces. What editor are you using? You can generally configure it to indent with different things for different file types.